packages/database/schema.prisma

```diff
@@ -459,6 +459,9 @@ model User {
   name                      String?
   email                     String           @unique
   emailVerified             DateTime?        @map(name: "email_verified")
+  twoFactorSecret           String?
+  twoFactorEnabled          Boolean          @default(false)
+  backupCodes               String?
```

Comment:
Is it secure enough to store the backup codes and twoFactorSecret in the database even though it's encrypted? Or would be hashing (like a password or api key) be a better approach?

(But otherwise they are generated by us so there is no additional security to the user (like with a password) and when the attacker has the database content, they also have all the survey & response data also, so no real difference to having access to the account)